sevsnpmeasure/sev_mode.py

```diff
@@ -19,5 +20,7 @@ def from_str(s: str):
             return SevMode.SEV_ES
         elif s == "snp" or s == "sev-snp" or s == "SNP" or s == "SEV-SNP":
             return SevMode.SEV_SNP
+        elif s.lower() == "snp+svsm" or s == "sev-snp+svsm":
```

Comment:
Should we add `.lower()` on the right-hand equality too?

Reply:
Done